zipline/assets/_securities.pyx

```diff
@@ -0,0 +1,189 @@
+#
+# Copyright 2015 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Cythonized Security object.
+"""
+cimport cython
+
+import numpy as np
+cimport numpy as np
+
+
+# IMPORTANT NOTE: You must change this template if you change
+# Security.__reduce__, or else we'll attempt to unpickle an old version of this
+# class
+CACHE_FILE_TEMPLATE = '/tmp/.%s-%s.v3.cache'
```

Comment:
This might be able to be removed?

It's an artifact of how we do caching, but might want to remove it until/unless there is an eventual solution in Zipline.